### PR TITLE
Experiment/active learning validation

### PR DESCRIPTION
**Pull request by noa-weiss**

### CODE REVIEW

File: datasets/orcasound/adapt_active_learning_annotations.ipynb [hunk 1]
```diff
@@ -0,0 +1,1909 @@
+{
```
Comment (amitgalor18) [inline datasets/orcasound/adapt_active_learning_annotations.ipynb:1]: Is the whole notebook just one large dict\some data object?

**Review by amitgalor18 — APPROVED** [review]
Since it seems that the whole pr is just additional utility files for active learning I see no reason not to approve it

**Comment by noamzbr** [comment]
I lowkey believe that stuff that is in the repo has to be related to training and serving models, so that active learning related code does fit but the annotation produced by it belong more in s3/clickup/someotherstorage, but this is not a strongly held belief (:

**Comment by noa-weiss** [comment]
@noamzbr , I'm willing to go with the general opinion here. @mosheman5 , WDYT?

**Comment by noa-weiss** [comment]
> @noamzbr , I'm willing to go with the general opinion here. @mosheman5 , WDYT?

Paging Dr. @mosheman5

File: datasets/orcasound/adapt_active_learning_annotations.ipynb [hunk 2]
```diff
@@ -0,0 +1,1909 @@
...
+ "cells": [
+  {
+   "cell_type": "markdown",
```
Comment (mosheman5) [inline datasets/orcasound/adapt_active_learning_annotations.ipynb:4]: Can you please clear the outputs of the notebook before committing it to git? :) 
https://stackoverflow.com/questions/28908319/how-to-clear-jupyter-notebooks-output-in-all-cells-from-the-linux-terminal
you can also do it directly inside the notebook using the GUI

**Comment by noa-weiss** [comment]
Could you explain why?

**Comment by mosheman5** [comment]
> 

Generally speaking outputs are static objects, and they are leaves of the process.
Ie, they are not used in any other process down the line, and we need no diff logging for them (unlike metadata objects/csv that are needed for training a model).
The main disadvantage of saving output data is unnecessary enlarging the repo.
Ideally the code should be slim, and if additional data is generated/needed we put references to its location in the code instead of the data itself.

**Pull request merged**